yb-voyager/src/queryissue/queryissue.go

```diff
@@ -56,17 +56,36 @@ func (p *ParserIssueDetector) GetIssues(query string) ([]issue.IssueInstance, er
 				log.Errorf("error getting issues in query-%s: %v", query, err)
 				continue
 			}
-			for _, i := range issuesInQuery {
-				//Replacing the objectType and objectName to the original ObjectType and ObjectName of the PLPGSQL object 
-				//e.g. replacing the DML_QUERY and "" to FUNCTION and <func_name>
-				i.ObjectType = plpgsqlObjType
-				i.ObjectName = plpgsqlObjName
-				issues = append(issues, i)
-			}
+			issues = append(issues, issuesInQuery...)
+		}
+		return lo.Map(issues, func(i issue.IssueInstance, _ int) issue.IssueInstance {
+			//Replacing the objectType and objectName to the original ObjectType and ObjectName of the PLPGSQL object 
+			//e.g. replacing the DML_QUERY and "" to FUNCTION and <func_name>
+			i.ObjectType = objType
+			i.ObjectName = objName
+			return i
+		}), nil
+	}
+	//Handle the Mview/View DDL's Select stmt issues
+	if queryparser.IsViewObject(parseTree) || queryparser.IsMviewObject(parseTree) {
+		selectStmt := queryparser.GetSelectStmtFromViewOrMView(parseTree)
+		selectStmtQuery, err := queryparser.DeparseSelectStmt(selectStmt)
```

Comment:
can we call Deparse directly inside `GetSelectStmtFromViewOrMView` and have it return a string directly? So that this layer doesn't have to care about deparsing.